.github/workflows/e2e.yml

```diff
@@ -0,0 +1,40 @@
+name: ContractClient
+
+on:
+  push:
+    branches: [master, release/**]
```

Comment:
This is going to slow the PR cycle down significantly. I'd prefer a time-based approach, e.g. once a day, plus on a release for sanity; wdyt?

Reply:
As long as it isn't a requirement for merging it shouldn't slow down the cycle.